src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { applyMiddleware, combineReducers, createStore } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
```

Comment:
- [ ] Please in order to manipulate the `categories state`, you need to define the `categories reducer`, import it to `configureStore.js` and then add it to `combineReducers`, something like so; `import categoriesReducer from './categories/categories' ` and then you add it to `combineReducers`. Please kindly fix that. Please just a reminder, before defining the categories reducer, you need to define the `check_status` action types using the ducks pattern and don't forget to define the `action creators`.